src/engine/renderer/tr_backend.cpp

```diff
@@ -3216,18 +3214,26 @@ void RB_RenderSSAO()
 
 	if ( r_ssao->integer < 0 ) {
 		// clear the screen to show only SSAO
-		GL_ClearColor( 1.0f, 1.0f, 1.0f, 1.0f);
+		GL_ClearColor( tr.mapInverseLightFactor, tr.mapInverseLightFactor, tr.mapInverseLightFactor, 1.0 );
 		glClear( GL_COLOR_BUFFER_BIT );
 	}
 
 	gl_ssaoShader->BindProgram( 0 );
 
-	zParams[ 0 ] = 2.0f * tanf( DEG2RAD( backEnd.refdef.fov_x * 0.5f) ) / glConfig.vidWidth;
-	zParams[ 1 ] = 2.0f * tanf( DEG2RAD( backEnd.refdef.fov_y * 0.5f) ) / glConfig.vidHeight;
-	zParams[ 2 ] = backEnd.viewParms.zFar;
+	vec3_t zParams;
+	zParams[0] = 2.0f * tanf( DEG2RAD( backEnd.refdef.fov_x * 0.5f ) ) / glConfig.vidWidth;
```

Comment:
We do want the spaces inside square brackets. If the code has spaces insides parentheses it should have spaces inside brackets too

Reply:
Fixed.